pulseaudio-control.bash

```diff
@@ -16,8 +16,15 @@ MUTED_COLOR="%{F#6b6b6b}"  # Color when the audio is muted
 DEFAULT_SINK_ICON="# "  # The default sink icon if a custom one isn't found
 CUSTOM_SINK_ICONS=(  )  # Custom sink icons in index of sink order
 NOTIFICATIONS="no"  # Notifications when switching sinks if enabled
+NICKNAMES="no"  # Use human-readable names instead of sink indices
 SINK_BLACKLIST=(  )  # Index blacklist for sinks when switching between them
 
+# maps pulse-audio sink names to human-readable names
+declare -A NAME_NICKNAME_MAP=(
```

Comment:
Have you checked what happens in these cases?

* A nickname is set to an non-existing sink
* A sink doesn't have a nickname

Reply:
non-existing sink: doesn't matter, it's just an unnecessary map entry in that case
no nickname in map: fallback to "device.description" property